src/middlewares/User.js

```diff
@@ -30,7 +30,36 @@ const checkEmail = async (req, res, next) => {
   }
 };
 
+const userNameAvailability = async (req, res, next) => {
```

Comment:
The naming of this variable is not correct. it should be usernameAvailability, Username is one word.

Reply:
changed usernameCheck and usernameAvailability both. Thanks.